extension/js/common/browser/browser.ts

```diff
@@ -27,7 +26,7 @@ export class Browser {
     const blob = new Blob([attachment.getData()], { type: attachment.type });
     const a = window.document.createElement('a');
     a.href = window.URL.createObjectURL(blob);
-    a.download = Xss.escape(attachment.name);
+    a.download = attachment.name;
```

Comment:
@rrrooommmaaa FYI This escape function converted special characters to something like `&quot;` and as I checked it's not needed for `download` a tag.